PyInstaller/utils/hooks/qt.py

```diff
@@ -371,62 +373,164 @@ def qt_plugins_binaries(plugin_type, namespace):
 #
 _qt5_dynamic_dependencies_dict = {
     ## "lib_name":              (.hiddenimports,           translations_base,  zero or more plugins...)
-    "qt5bluetooth":             (".QtBluetooth",           None,               ),  # noqa: E241,E202
-    "qt5concurrent":            (None,                     "qtbase",           ),
-    "qt5core":                  (".QtCore",                "qtbase",           ),
+    "qt5bluetooth": (
```

Comment:
This is quite ugly, but it's hard to blame the formatter. 

We should probably rework to use named tuples with `hiddenimports`, `translation_base`, and `plugins` fields. Then entries can be split in multiple lines, and corresponding fields assigned via keyword arguments, and there will be no need for visual alignment of "columns" in the "table".